core/client/fs/src/main/java/alluxio/client/file/cache/store/RocksPageStore.java

```diff
@@ -76,14 +76,15 @@ public void put(PageId pageId, byte[] page) throws IOException {
   }
 
   @Override
-  public ReadableByteChannel get(PageId pageId) throws IOException,
-      PageNotFoundException {
+  public ReadableByteChannel get(PageId pageId, int pageOffset)
```

Comment:
Precondition to check page Offset is positive?

Reply:
Added.